apps/api/src/migration/1660225963147-AddDeleteCascadeToJobStatusAudit.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddDeleteCascadeToJobStatusAudit1660225963147 implements MigrationInterface {
+  name = 'AddDeleteCascadeToJobStatusAudit1660225963147';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "ien_applicant_status_audit" DROP CONSTRAINT "FK_ec2940cc497a06ba811763be464"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "ien_applicant_status_audit" ADD CONSTRAINT "FK_ec2940cc497a06ba811763be464" FOREIGN KEY ("job_id") REFERENCES "ien_applicant_jobs"("id") ON DELETE CASCADE ON UPDATE NO ACTION`,
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Is this right that `up` and `down` are same?

Reply:
up contains 'ON DELETE CASCADE', down contains 'ON DELETE NO ACTION'